package.json

```
@@ -1,62 +1,65 @@
 {
-  "name": "@matrix-org/react-sdk-module-api",
-  "version": "2.1.0",
-  "description": "Module API surface for matrix-react-sdk",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
-  "repository": "git@github.com:matrix-org/matrix-react-sdk-module-api.git",
-  "author": "The Matrix.org Foundation C.I.C.",
-  "license": "Apache-2.0",
-  "files": [
-    "lib",
-    "README.md",
-    "package.json",
-    "yarn.lock",
-    "LICENSE"
-  ],
-  "scripts": {
-    "clean": "rimraf lib",
-    "build": "yarn clean && yarn build:compile && yarn build:types",
-    "build:types": "tsc -p ./tsconfig.build.json --emitDeclarationOnly",
-    "build:compile": "babel -d lib --verbose --extensions \".ts,.tsx\" src",
-    "start": "tsc -p ./tsconfig.build.json -w",
-    "test": "jest",
-    "lint": "eslint src test && tsc --noEmit",
-    "lint:fix": "eslint --fix src test",
-    "prepublishOnly": "yarn build"
-  },
-  "devDependencies": {
-    "@babel/cli": "^7.17.6",
-    "@babel/core": "^7.17.8",
-    "@babel/eslint-parser": "^7.17.0",
-    "@babel/eslint-plugin": "^7.17.7",
-    "@babel/plugin-proposal-class-properties": "^7.16.7",
-    "@babel/plugin-transform-runtime": "^7.17.0",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-react": "^7.16.7",
-    "@babel/preset-typescript": "^7.16.7",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^12.1.5",
-    "@types/jest": "^27.4.1",
-    "@types/react": "^17",
-    "@types/react-dom": "^17.0.19",
-    "@typescript-eslint/eslint-plugin": "^5.18.0",
-    "@typescript-eslint/parser": "^5.18.0",
-    "eslint": "^8.12.0",
-    "eslint-config-google": "^0.14.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-matrix-org": "^0.4.0",
-    "jest": "^27.5.1",
-    "react": "17.0.2",
-    "react-dom": "^17.0.2",
-    "rimraf": "^3.0.2",
-    "ts-jest": "^27.1.4",
-    "typescript": "^4.6.3"
-  },
-  "dependencies": {
-    "@babel/runtime": "^7.17.9"
-  },
-  "peerDependencies": {
-    "react": "^17.0.2"
-  }
+    "name": "@matrix-org/react-sdk-module-api",
+    "version": "2.1.0",
+    "description": "Module API surface for matrix-react-sdk",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "repository": "git@github.com:matrix-org/matrix-react-sdk-module-api.git",
+    "author": "The Matrix.org Foundation C.I.C.",
+    "license": "Apache-2.0",
+    "files": [
+        "lib",
+        "README.md",
+        "package.json",
+        "yarn.lock",
+        "LICENSE"
+    ],
+    "scripts": {
+        "clean": "rimraf lib",
+        "build": "yarn clean && yarn build:compile && yarn build:types",
+        "build:types": "tsc -p ./tsconfig.build.json --emitDeclarationOnly",
+        "build:compile": "babel -d lib --verbose --extensions \".ts,.tsx\" src",
+        "start": "tsc -p ./tsconfig.build.json -w",
+        "test": "jest",
+        "lint": "eslint src test && tsc --noEmit && prettier --check",
```

Prettier needs a argument what to check:

```suggestion
        "lint": "eslint src test && tsc --noEmit && prettier --check .",
```